org.eclipse.winery.repository.rest/pom.xml

```diff
@@ -209,6 +209,23 @@
             <artifactId>org.eclipse.winery.common</artifactId>
             <version>2.0.0-SNAPSHOT</version>
         </dependency>
+        <!-- begin of EDMM dependencies -->
+        <dependency>
+            <groupId>org.eclipse.winery</groupId>
+            <artifactId>org.eclipse.winery.edmm</artifactId>
+            <version>2.0.0-SNAPSHOT</version>
+        </dependency>
+        <dependency>
+            <groupId>org.jgrapht</groupId>
+            <artifactId>jgrapht-core</artifactId>
+            <version>1.1.0</version>
```

Comment:
make sure you use the same version as Winery currently uses.

Reply:
I have tried using the version specified in the root pom.xml file (1.3.1) but it fail giving as error:
`java.lang.NoClassDefFoundError: org/jgrapht/EdgeFactory`

Reply:
Solve this, please. Check where the NoClassDefFoundError occurs and migrate it. I suggest du try using the latest possible version (probably the one that comes with the transformation framework)

Reply:
The transformation-framework used the version `1.1.0`, the one I putted here.

Reply:
Then upgrade the transformation framework to have it working with 1.3.1. It's a MUST HAVE to align these versions.

Reply:
this should be solved. I updated the version of jgrapht on the transformation-framework and setted here the version used by winery.